test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -2882,97 +2873,126 @@ TEST_F(ControlPlaneStatusRewritingTest, RewritesFromConfigSelector) {
 // WeightedRoundRobinTest
 //
 
-using WeightedRoundRobinTest = ClientLbEnd2endTest;
+const char kServiceConfigPerCall[] =
+    "{\n"
+    "  \"loadBalancingConfig\": [\n"
+    "    {\"weighted_round_robin_experimental\": {\n"
+    "      \"blackoutPeriod\": \"0s\",\n"
+    "      \"weightUpdatePeriod\": \"0.1s\"\n"
+    "    }}\n"
+    "  ]\n"
+    "}";
+
+const char kServiceConfigOob[] =
+    "{\n"
+    "  \"loadBalancingConfig\": [\n"
+    "    {\"weighted_round_robin_experimental\": {\n"
+    "      \"blackoutPeriod\": \"0s\",\n"
+    "      \"weightUpdatePeriod\": \"0.1s\",\n"
+    "      \"weightExpirationPeriod\": \"2s\",\n"
```

Comment:
I don't think we need to set this field, since none of the tests depend on it.

Reply:
Done